packages/forms-web-app/src/lib/selected-appeal-page-setup.js

```diff
@@ -60,12 +64,36 @@ const getFinalComments = (caseData, isAppellantComments) => {
 
 //Statements
 /**
- * @param {string} userType
+ * @param {string} url
  * @returns {string}
  */
-const formatStatementHeading = (userType) => {
-	if (userType === LPA_USER_ROLE) return 'Your';
-	return 'Local planning authority';
+const formatStatementHeading = (url) => {
+	if (url.includes('other-party-statements')) {
+		return 'Statements from other parties';
+	} else if (url.includes('lpa-statement')) {
+		return 'Local planning authority statement';
+	} else {
+		return 'Your statement';
+	}
+};
+
+/**
+ * @param {string} url
+ * @param {AppealUser} user
+ * @returns {string}
+ */
+const getStatementType = (url, user) => {
```

Comment:
Couldn't there be an statement from appellant/agent?
Can we const these types as well rather than relying on the string value